doc/devel/document.rst

```diff
@@ -1053,6 +1051,54 @@ subdirectory, but  :file:`galleries/users_explain/artists` has a mix of
 any ``*.rst`` files to a ``:toctree:``, either in the ``README.txt`` or in a
 manual ``index.rst``.
 
+Examples guidelines
+-------------------
+
+The gallery of examples contains visual demonstrations of matplotlib features. Gallery
+examples exist so that users can scan through visual examples. Unlike tutorials or user
+guides, gallery examples teach by demonstration, rather than by explanation or
+instruction.
+
+Gallery examples should contain a very brief description of *what* they demonstrate and,
+when relevant, *how* it is achieved. Avoid instructions or excessive explanation;
+instead, tag with related concepts and cross-link to relevant tutorials or user guides.
+
+Format
+^^^^^^
+
+All :ref:`examples-index` should aim to follow these guidelines:
+
+:Title: Describe content in a short sentence (approx. 1-6 words). Do not use *demo* as
```

Comment:
Does sentence imply a full sentence including verb? Possible alternatives: expression, phrase ... (not a native speaker so cannot really judge)